### DISCUSSION TITLE
Data Loaders in Nuxt

### DISCUSSION THREAD

**Discussion by sergenux** [posva/unplugin-vue-router#409] [Questions and Help] [answered]
Hi, I'm using Nuxt 3 and add `typedPages: true` option in experimental section of `nuxt.config.js`
How can i enable DataLoaderPlugin in Nuxt?
Thanks for any help.

**Comment by posva** [comment, marked as answer]
I did adapt the data loaders to be independent from uvr. ~~I'm in the process of writing a Nuxt module to provide a smooth integration with Data Loaders independently from the typedPages option.~~

Given the current architecture, it turned out to be better to have [a Nuxt plugin](https://uvr.esm.is/data-loaders/nuxt.html)

Currently you can manually create a plugin that ensures the [installation of Data Loaders](https://uvr.esm.is/data-loaders/#installation)

**Reply by husayt** [reply]
Hi @posva where can we find out about this module?

Thank you for leading this great effort

**Reply by posva** [reply]
It hasn't been published yet. You can still currently write a plugin like in the docs
